js-miniapp-sdk/src/types/responseTypes/rewarded/index.ts

```diff
@@ -0,0 +1,8 @@
+import { AdTypes } from '../../adTypes';
+/**
+ * A contract declaring the interaction mechanism between Rewarded ad type response from native SDK
+ */
+export interface RewardedAdResponse {
+  amount?: number;
+  adType: AdTypes.REWARDED;
+}
```

Comment:
Rewarded ads also have a reward type: https://developers.google.com/android/reference/com/google/android/gms/ads/rewarded/RewardItem#getType()

So maybe the response should be something like this?

```suggestion
export interface Reward {
  amount: number;
  type: string
}

export interface RewardedAdResponse {
  reward: Reward?;
  adType: AdTypes.REWARDED;
}
```

Reply:
Added new reward interfae!